__tests__/components/next_steps_test.js

```diff
@@ -2,59 +2,55 @@ import React from "react";
 import { NextSteps } from "../../components/next_steps";
 import { mount } from "enzyme";
 import translate from "../fixtures/translate";
+import nextStepsFixture from "../fixtures/nextSteps";
 
 const { axe, toHaveNoViolations } = require("jest-axe");
 expect.extend(toHaveNoViolations);
 
 describe("NextSteps", () => {
   let props;
-  let reduxState;
   beforeEach(() => {
     props = {
       t: translate,
-      mapUrl: "/map"
+      mapUrl: "/map",
+      nextSteps: nextStepsFixture
     };
-
-    reduxState = {};
-    props.reduxState = reduxState;
   });
 
   it("passes axe tests", async () => {
-    let html = mount(<NextSteps {...props} {...reduxState} />).html();
+    let html = mount(<NextSteps {...props} />).html();
```

Comment:
we don't actually need to pass in reduxState here since we aren't mocking it b/c there are not child components that use redux